### ISSUE TITLE
Ability to view migrated Works in Hyrax

### ISSUE THREAD

**Issue by abelemlih** [emory-libraries/OpenEmory#150]
For basic read-only testing in Hyrax against our migrated data, we should verify if the following can function:

- [x] A search search/browse in the Hyrax front end returns indexed OpenEmory work records
- [x] An individual work record and its indexed metadata can be viewed

**Additional notes**

The following fields are required for works in Hyrax: (https://samvera.github.io/metadata_application_profile.html)

* creator
* rights_statement 
* title
* depositor
* date_uploaded
* date_modified

If useful, here's a[ basic mapping](https://docs.google.com/spreadsheets/d/1zyeGNv1UF9ddBn3OA1WMLebiul8kU6hxU7X2k88HfKE/edit?usp=sharing) of what we already have been indexing for SOLR.

**Comment by eporter23** [comment]
@abelemlih if this is helpful, I started working on this after @bwatson78 reported some immediate SOLR errors: https://docs.google.com/spreadsheets/d/1zyeGNv1UF9ddBn3OA1WMLebiul8kU6hxU7X2k88HfKE/edit?usp=sharing

**Comment by abelemlih** [comment]
PR ready for review: https://github.com/emory-libraries/fedora6-hyrax-prototype/pull/7

@eporter23 I will deploy the changes in this PR to the Hyrax instance once a Solr 8 instance is available. I discussed with Beth setting up the Solr 8 instance, since Hyrax is only compatible with Solr up to 8.11.

The changes in the PR above add the following to the prototype:
- Ability to index all works in the Fedora 6 prototype as Generic Works in the Hyrax instance
- Ability to open each work and view its associated metadata. For now, the metadata available are: `title`, `description`, `date_created`, `updated_at`, `creator`, `contributor`, `subject`, `language`, `identifier`, `source`
- 

Please refer to the screenshots attached, and let me know if you have any questions. I will let you know once the Hyrax prototype instance is ready for testing.

![Screenshot 2023-07-06 at 5.03.07 PM.png](https://images.zenhubusercontent.com/6151f04e7e6954612001d8f9/c529ea26-cad8-411a-82e7-1bb72519b4ca)

![Screenshot 2023-07-06 at 5.03.32 PM.png](https://images.zenhubusercontent.com/6151f04e7e6954612001d8f9/2b86ef3c-ec57-41d7-aad7-9c3443259aaf)

![image.png](https://images.zenhubusercontent.com/6151f04e7e6954612001d8f9/87b765a1-02f1-49da-b900-14665b4305bd)

**Comment by abelemlih** [comment]
@eporter23 all changes have been deployed to the Hyrax prototype sandbox. Let me know once this ticket is ready for testing, so we discuss how to authenticate to view all the works.

**Comment by eporter23** [comment]
@abelemlih this is looking great! I can do a browse/empty search but noticed that I can't match against any specific search terms. If that should be a separate ticket, let me know. Or, does this have to do with the SOLR 8 version?

**Comment by abelemlih** [comment]
I discussed the issue above with Emily and I will address it in a separate ticket.
